promises.cf

```diff
@@ -57,6 +57,10 @@ body common control
 
       version => "CFEngine Promises.cf 3.7.0";
 
+      cfengine_3_7.(debian||redhat)::
```

Comment:
Indeed, I think ``||`` won't work actually, it will think you are ORing with a class called literally ``|redhat``.